plugins/grounded/grounded.cpp

```diff
@@ -0,0 +1,177 @@
+#include "ProcessWindows.h"
+#define MUMBLE_PLUGIN_NO_DEFAULT_FUNCTION_DEFINITIONS

```

Comment:
why? Unless you are including the plugin header multiple times, having the default definitions saves you the trouble of having to implement some functions yourself :thinking:

Reply:
I think I just copied this from elsewhere to try to be conventional, but I think you're right in that it doesn't seem appropriate here.